Cargo.toml

```diff
@@ -30,14 +30,13 @@ lazy_static = "1.4"
 ouroboros = "0.15.0"
 
 [dev-dependencies]
-lettre = "0.9.2"
-lettre_email = "0.9.2"
+lettre = "0.10"
 rustls-connector = "0.16.1"
 structopt = "0.3"
 
 # to make -Zminimal-versions work
 encoding = "0.2.32"
-hostname = "0.1.3"
+hostname = "0.3.1"
```

Comment:
This specifier was _only_ there to make `-Zminimal-versions`, so if bumping the major version here is now okay, it can probably be removed entirely!

Reply:
removed.